File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
def pad(array, min_size, value = None): 
    size = min_size - len(array)
    result = []
    if size <= 0:
        return array
    else: result = [value] * min_size
    for i,j in enumerate(array):
        result[i] = j
    return result








#       pseudocode

# first we need to obtain the input arrays length, and compare it with the min size. If it's smaller return the original array.

# second we create a result array of the size asked in the input filled with the value asked.

# last we replace the result values with the input array ones.

File: js/padArray.js
// REMEMBER TO PSEUDOCODE
exports.padArray = (array, minSize, value=null) => {
    const diff = minSize - array.length
    if (diff <= 0){
        return array
    }
    result = new Array(minSize).fill(value)
    
    for (let i = 0; i < array.length; i++){
        result[i] = array[i]
    }
    
    return result
}
/* first get the size of the array versus the minSize, if it's under 1 return the input array

second create an array of the minSize length filled with the value asked

last we assing the values of the input array into the result array*/